### PR TITLE
PROJECT SUMMARY TR-EN var-1

### LINKED ISSUE #52: PROJE ÖZETİ - English

### PR DESCRIPTION
**Pull request by ahmetbir**
First variation of Summery.

### CODE REVIEW

File: docs/TUBITAK-doc/ProjeÖzeti.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+ - It allows students to learn not only by doing homework, but also by checking homework,
+
+ - It ensures the fair distribution of students' grades.
+
+
+
+
+
+SOTAS, öğrencilerin ödev yapmasını ve diğer arkadaşlarının ödevlerini değerlendirerek puanlayıp neden o puanı verdiğine dair yorum yapmasını sağlayan bir platformdur.
```
Comment (berkaycubuk) [inline docs/TUBITAK-doc/ProjeÖzeti.md:57]: What this text for?

**Review by berkaycubuk — APPROVED** [review]
Whatever don't answer my questions but in someday you'll wish you answered :smiling_imp:

File: docs/TUBITAK-doc/ProjeÖzeti.md (hunk 1 above)
Comment (First008) [reply docs/TUBITAK-doc/ProjeÖzeti.md:57]: Imposter turkish sentence just looking around and investigating english sentences.

Comment (ervacoban) [reply docs/TUBITAK-doc/ProjeÖzeti.md:57]: when the imposter is sus

**Pull request merged**